src/App.tsx

```diff
@@ -2,13 +2,27 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import { PostList } from './components/PostList/PostList';
+
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts = posts.map((post) => {
+  const userLink = users.find(user => post.userId === user.id);
+  const commentsArray = comments.filter(comment => post.id === comment.postId);
+
+  return {
+    user: userLink || null,
+    comments: commentsArray || null,
```

Comment:
It'll never be null, cause empty array is still == `true` when converted to boolean
```suggestion
    comments: commentsArray,
```